stylesheet.css

```diff
@@ -0,0 +1,36 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+
+body {
+  background-color: #181818;
+  width: 100%;
+  height: 100vh;
```

Comment:
A body should have `min-height`, not `height`.
This is because when you have too much content in the body then it will not expand.